js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -27,70 +71,197 @@ export const TickDetailsDialog: React.FC<{
-          ) : tick.status === InstigationTickStatus.FAILURE && tick.error ? (
-            <PythonErrorInfo error={tick.error} />
-          ) : undefined}
-        </DialogBody>
+    <>
+      <DialogHeader
+        label={
+          <TimestampDisplay
+            timestamp={tick.timestamp}
+            timeFormat={{showTimezone: false, showSeconds: true}}
+          />
+        }
+      />
+      <Box padding={{vertical: 12, horizontal: 24}} border="bottom">
+        <TickDetailSummary tick={tick} />
+      </Box>
+      {tick.runIds.length || tick.originRunIds.length ? (
+        <>
+          <Box padding={{vertical: 12, horizontal: 24}} border="bottom">
+            <Subtitle2>Runs requested</Subtitle2>
+          </Box>
+          {tick.runIds.length ? (
+            <RunList runIds={tick.runIds} />
+          ) : (
+            <FailedRunList originRunIds={tick.originRunIds} />
+          )}
+        </>
       ) : null}
-      <DialogFooter>
-        <Button
-          icon={<Icon name="copy_to_clipboard" />}
-          onClick={(e) => copyValue(e, window.location.href)}
-        >
-          Copy Link
-        </Button>
-        <Button intent="primary" onClick={onClose}>
-          OK
-        </Button>
-      </DialogFooter>
-    </Dialog>
+      {addedPartitions?.length ? (
+        <>
+          <Box padding={{vertical: 12, horizontal: 24}} border="bottom">
+            <Subtitle2>Added partitions</Subtitle2>
+          </Box>
+          <PartitionsTable partitions={addedPartitions} />
+        </>
+      ) : null}
+      {deletedPartitions?.length ? (
+        <>
+          <Box padding={{vertical: 12, horizontal: 24}} border="bottom">
+            <Subtitle2>Deleted partitions</Subtitle2>
+          </Box>
+          <PartitionsTable partitions={deletedPartitions} />
+        </>
+      ) : null}
+    </>
   );
 };
 
+export function TickDetailSummary({tick}: {tick: HistoryTickFragment | AssetDaemonTickFragment}) {
+  const [showPythonError, setShowPythonError] = React.useState<PythonErrorFragment | null>(null);
+
+  const intent = React.useMemo(() => {
+    switch (tick?.status) {
+      case InstigationTickStatus.FAILURE:
+        return 'danger';
+      case InstigationTickStatus.STARTED:
+        return 'primary';
+      case InstigationTickStatus.SUCCESS:
+        return 'success';
+    }
+    return undefined;
+  }, [tick]);
+
+  const isAssetDaemonTick = 'requestedAssetMaterializationCount' in tick;
+
+  return (
+    <>
+      {showPythonError ? (
+        <Dialog
+          title="Error"
+          isOpen={!!showPythonError}
+          onClose={() => {
+            setShowPythonError(null);
+          }}
+          style={{width: '80vw'}}
+        >
+          <DialogBody>
+            <PythonErrorInfo error={showPythonError} />
+          </DialogBody>
+          <DialogFooter topBorder>
+            <Button
+              intent="primary"
+              onClick={() => {
+                setShowPythonError(null);
```

Comment:
I think it'd be preferable to use showCustomAlert for this - seems like it's /almost/ the same as that version but is missing the copy/paste button. Not sure if this was necessary because it's appearing within another dialog? I think we use showCustomAlert inside the launchpad modal. 

eg:
```     
 showCustomAlert({
        title: 'Unable to load tags',
        body: <PythonErrorInfo error={partition.tagsOrError} />,
      });
```

Reply:
good call